examples/h2_server/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC4fLPWRShlq+o6
+vM5nhUCtR+INmlAlQCzM3asWpQ76T/MOX1Ci1brypRgOXCmYI8c5lFGOIMpH1ZAh
+987t4UdkwVCn76pXv5yY4U1jJ5i9v7R7IUeKZ7utwlq1jPo3WWhiZwm1o6AxVRHG
+iUQ5XrcX0Rgt0bh9/5BWHLks3OJ44myjpgGv2J7n1LcyecRAd+suN4qsBiTITQBY
+q27WYsDCqrVBBoyCPhb51xTlaMlcAH2ekrm886FlC95VrwV7o4jb4sRGYdfKqT6H
+wXK1yTB3n7422WavDgRj2vNQaL+XoPiBOrQZ8fj9PeWCyefSEtqz8/dpnJO+pxid
+KAow8As5AgMBAAECggEAF2SIkSTXyHySCLykF3BIETDKMxtcCj0m8RsydH6ddZPp
+p+riGKOtPtM8R6XcCFSREHYEGVeDDrS/v+7iKDumh9oTVhgHq3S6cJAw9d+xeG30
+fqBrp9sbax73zYmAuL/3gnK6QnPq/hZhWKFTK0sdQEHdJQIwuJlNPf4KZOvlM5qa
+1xSL1k0q1vFPaX7yINuLtZoJTOes4Nnxgyf/lGETdeikG4KZ5Pw/sSElId7VMcRw
+Jxe2UKFvtsxTsMym9Zin1cEzgJDY+l0c3nFu8o/w/ogfROmN0Gt0TXrI70/Xixom
+jLUZI7HzLrNTFMx2DIVSdcrhsL6dSLdVGupyKRHcYQKBgQD0xSwYCFUXwJFSyC9u
+f+rVAzWgUbUOPX4tgSddc1t066BlWPYvx5OiUAUWOmtyJ6YAOaqGfE2AzQS/5FVn
+NINkWsCFD41grDolB78R8i+cbK3kZeegglDiGyJtEfZQMegbPFZPBzFs50zeaqdj
+dYIwDclagsMSoCkZEOH+EUVi1QKBgQDA84FHJ/HnBOD8czaRpUROA+QwvzU+8Z0n
+GAoNbpdpsRV2p/cOzVR8QsdEIrQsfRFL7wvkYMG4F4C6Gle3OSbArfStMx9WrGLs
+iV11xikGoKD1sd3KDqIAjaG3Cb6P1otII/C414JKohgoi8uty+sVY6UtFiidLGXK
+PtcsXKmQ1QKBgCeOQaz0k8PN4+5L2ZZy56Fa7mylJ6xQdbam25S3kr0XYwld0hMw
+noQpiFjqQfjlvWbl6s/nAKNi/nSAUujlzKIIJHqcY82Op6raUJ920wjla0iwyzje
+NnIDzpFvyK13Ezprisskr3EZBAwhkkF5ePuJDtQos5ur9FhPqSXk5qCtAoGAYux1
+G0Yff5HAx61KDOHuYxEbrOJbKio1qBVrEffhKQ7hMjRrNeuPpSWXhBdGyIhWlde5
+AR2aNAfYt5FJ52YluCN3jlTnfEhlb1/9yqQN5Jf17gGC9FUMObxjbLRSPux//Dxw
+7lOPOTRhrmf9xN0Haa/sv3Dc4bghd89AAj5/sQECgYEAqC8iD9u8XvTkuDZt5uNH
+YxV5XJINk/9KNjmbL7hwXxVIk4r7xYDUx4l2/iy6QaMAyswCl6AqZqIieBvaLMg1
+9qcR3TGfiHWcLBVxFo2v19KRkMGGL4kSuiXSNqee2+YHJc8PQFeC7Nrs1Ukaa3ML
+JmT6s4e0LUn6obuR1cek9hM=
+-----END PRIVATE KEY-----
```

Comment:
I'd rename those as something like `server_insecure.key` (what Vagrant is doing), just so it's wrll clear, same for the crt